packages/flex-plugin-e2e-tests/src/utils/pages/view/base.ts

```diff
@@ -1,14 +1,21 @@
 import { ElementHandle, Page } from 'puppeteer';
+import { logger } from '@twilio/flex-dev-utils';
+
+const LOCATE_TIMEOUT: string | undefined = process.env.LOCATE_TIMEOUT;
+const PAGE_LOAD_TIMEOUT: string | undefined = process.env.PAGE_LOAD_TIMEOUT;
 
 export abstract class Base {
-  protected static readonly DEFAULT_LOCATE_TIMEOUT = 60000;
+  protected static readonly DEFAULT_LOCATE_TIMEOUT = LOCATE_TIMEOUT ? Number(LOCATE_TIMEOUT) : 60000;
 
-  protected static readonly DEFAULT_PAGE_LOAD_TIMEOUT = 60000;
+  protected static readonly DEFAULT_PAGE_LOAD_TIMEOUT = PAGE_LOAD_TIMEOUT ? Number(PAGE_LOAD_TIMEOUT) : 60000;
```

Comment:
Lets keep this static and not read from env variables

Reply:
Keeping it 1 mins, is causing mac build failure with latest puppeteer version, Should I bump to 10 mins here as default ? Note: This is only needed for JS, not for TS.

Reply:
Yeah if macos needs a minimum of 10mins, then lets make that default. Please see the minimum default time it needs. I am guessing even 2-3 mins may work. Can we try once using the env variable and then change the code?

Reply:
Let me set 5 mins in code as default, If it fails will bump it to 10 mins. Finding exact time hard, as it may need multiple round of runs.

Reply:
Sounds good, thanks

Reply:
But having this config values in code, doesn't seem right, There should be flexibility to control these from outside.

Reply:
Yeah that makes sense for timeouts in general but here if page navigation/element locating is taking longer than 5 mins or whatever the default value is, I think we should fix the e2e flow in code or look into some issue that might be causing it rather than bumping up the timeout each time it fails. Practically a navigation exceeding 5 mins is definitely an issue that needs to be checked..